common/data_refinery_common/models/models.py

```diff
@@ -1137,9 +1154,13 @@ class Dataset(models.Model):
     # Processing properties
     aggregate_by = models.CharField(max_length=255, choices=AGGREGATE_CHOICES, default="EXPERIMENT", help_text="Specifies how samples are [aggregated](http://docs.refine.bio/en/latest/main_text.html#aggregations).")
     scale_by = models.CharField(max_length=255, choices=SCALE_CHOICES, default="NONE", help_text="Specifies options for [transformations](http://docs.refine.bio/en/latest/main_text.html#transformations).")
-    quantile_normalize = models.BooleanField(default=True, help_text="Part of the advanced options. Allows [skipping quantile normalization](http://docs.refine.bio/en/latest/faq.html#what-does-it-mean-to-skip-quantile-normalization-for-rna-seq-samples) for RNA-Seq samples.")
+    quantile_normalize = models.BooleanField(
+        default=True,
+        help_text="Part of the advanced options. Allows [skipping quantile normalization](http://docs.refine.bio/en/latest/faq.html#what-does-it-mean-to-skip-quantile-normalization-for-rna-seq-samples) for RNA-Seq samples."
+    )
     quant_sf_only = models.BooleanField(default=False, help_text="Include only quant.sf files in the generated dataset.")
 
+    svd_algorithm = models.CharField(max_length=255, choices=SVD_ALGORITHM_CHOICES, default="NONE", help_text="Specifies choice of SVD algorithm")
```

Comment:
The whitespace here and in 1157 above is different - not sure which style we prefer.

Reply:
👍 moved the space to below the line